src/pages/ScholarshipForm.jsx

```diff
@@ -20,20 +20,15 @@ function ScholarshipForm() {
     return (e) =>
       setFormFieldStates({ ...formFieldStates, [id]: e.target.value });
```

Comment:
btw I think `[id]` here can simply be `id`

Reply:
I tried it and what ends up happening is that the value doesn't update.

Reply:
Ah I think I know why, it's because it would set the `id` property instead of whatever the value of `id` is. That's why `[id]` is needed in this case.